### DISCUSSION TITLE
for help，How to click on every element in a list，TKS

### DISCUSSION THREAD

**Discussion by techflag** [AutomaApp/automa#349] [Q&A]
How to click on every element in a list
![image](https://user-images.githubusercontent.com/939362/154082371-e5b7d471-f991-4282-90f7-0107020c5b13.png)

**Comment by Jooee1** [comment]
I find a demo from automa marketplace.

Use loop data to loop the element and remember the ID,
![Snipaste_1](https://user-images.githubusercontent.com/115023248/197399307-0ede734d-6a87-4dda-8115-602f23a37a8c.jpg)

To add a click element, type `{{loopData@ID}}` (I guess it's a built-in variable), and insert your loop operation in the next step
![Snipaste_2](https://user-images.githubusercontent.com/115023248/197399326-dfb89948-e51a-4516-abd7-9b507654c550.jpg)
